src/elements/content-picker/Footer.scss

```diff
@@ -9,11 +9,23 @@
     justify-content: space-between;
     padding: 0 20px;
 
+    .bcp-footer-left {
+        display: flex;
+        flex: 1;
+        min-width: 0;
+    }
+
     .bcp-selected {
+        flex: 0 0 75px;
         margin: 0;
+        width: 100%;
 
         .btn-content {
-            display: flex;
```

Comment:
Is everything inside the selected button, including the red file limit text (`(2 max)`), still vertically centered with this change?

Reply:
looks like the centering is off, I'll look for a solution.